inc/Engine/Optimization/Minify/JS/Minify.php

```diff
@@ -183,12 +183,10 @@ protected function replace_url( $url ) {
 		}
 
 		// This filter is documented in /inc/classes/optimization/class-abstract-optimization.php.
-		$url       = apply_filters( 'rocket_asset_url', $url, $this->get_zones() );
-		$unique_id = md5( $url . $this->minify_key );
-		$filename  = preg_replace( '/\.js$/', '-' . $unique_id . '.js', ltrim( rocket_realpath( wp_parse_url( $url, PHP_URL_PATH ) ), '/' ) );
-
+		$url           = apply_filters( 'rocket_asset_url', $url, $this->get_zones() );
+		$filename      = ltrim( rocket_realpath( wp_parse_url( $url, PHP_URL_PATH ) ), '/' );
```

Comment:
Could you add the same checks we do on the URL in the CSS/Minify to this one? the code between line 126-135. It could help avoid some errors.